pubspec.yaml

```
@@ -0,0 +1,52 @@
+name: deviflix
+description: A new Flutter project.
+
+publish_to: "none"
+
+version: 1.0.0+1
+
+environment:
+    sdk: ">=2.19.6 <3.0.0"
```

Dart 3 sdk 를 사용해주셔야 위 리뷰에 있는 switch expression 과 같은 최신 문법 사용이 가능합니다. 새로운 기능 공부를 위해 변경 해보시면 좋을 것 같습니다!
```suggestion
    sdk: "3.2.3"
```

---

sdk 버전은 신경안쓰고 fvm만 쓸데없이 매번 업데이트 시켰었군요. 
어쩐지 Dart SDK관련 warning이 뜨더라니
`sdk: ">=3.2.3 <4.0.0"`요렇게 수정했습니다